dags/utils/gcp.py

```diff
@@ -711,14 +713,18 @@ def gke_command(
         namespace=gke_namespace,
         image=docker_image,
         arguments=command,
-        env_vars={
+        env_vars={**{
             key: value
             for key, value in zip(
                 ("AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"),
                 AwsHook(aws_conn_id).get_credentials() if aws_conn_id else (),
             )
-            if value is not None
-        },
+            if value is not None},
+            **{
+                "XCOM_PUSH": xcom_push
```

Comment:
we need to lower case this, json seems like the most succinct way to do that
```suggestion
                "XCOM_PUSH": json.dumps(xcom_push),
```

Reply:
i see now that it's checked uppercased, please ignore